client/src/style.scss

```diff
@@ -93,13 +48,13 @@ h5 {
   padding: 3px;
 }
 
-.persons-container {
+.persons-container, .date-container {
```

Comment:
This is okay if you think one modal will diverge from the other in the future. Otherwise if you think they will be the same, I personally would use a single, more generic classname like `container`, `modal` etc